### PR TITLE
Add highlights for the week ending on 2022-06-17

### PR DESCRIPTION
**Pull request by weatherhog**
Please add your contributions until June 17, 13:00 CEST

### CODE REVIEW

File: src/content/changes/highlights/2022-06-17.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2022-06-17T10:00:00
+title: Highlights for the week ending June 17, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
```
Comment (weatherhog) [inline src/content/changes/highlights/2022-06-17.md:9]:
```suggestion
With [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) release [v2.12.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2121---2022-06-09) we updated the controller image to fix the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8686) which fixes the unpublished `CVE-2021-25748`.
It is highly recommended to update all [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) installations.
```

File: src/content/changes/highlights/2022-06-17.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## General
+
+## User interfaces
+
```
Comment (kuosandys) [inline src/content/changes/highlights/2022-06-17.md:13]:
```suggestion

With `kubectl gs` release [v2.14.0](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.14.0), the service priority cluster label can be set when using the `template cluster` command with the `--service-priority` flag.

You can also set the service priority label during cluster creation via the web UI, and inspect cluster labels in their raw format from the cluster details page.

```
@anvddriesch @gusevda please edit as you see fit 🙂

Comment (anvddriesch) [reply src/content/changes/highlights/2022-06-17.md:13]: LGTM

**Review by MarcelMue — APPROVED** [review]
Nothing from honeybadger this week.

File: src/content/changes/highlights/2022-06-17.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+With [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) release [v2.12.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2121---2022-06-09) we updated the controller image to fix the upstream [issue](https://github.com/kubernetes/ingress-nginx/issues/8686) which fixes `CVE-2021-25748`.
+It is highly recommended to update all [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) installations.
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-06-17.md:13]:
```suggestion

## Apps
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.10.1](https://github.com/giantswarm/kyverno-app/compare/v0.10.0...v0.10.1) updates the CRDs installed by Kyverno to match the upstream version 1.6.2 CRDs.
- [security-pack](https://github.com/giantswarm/security-pack) version [v0.3.1](https://github.com/giantswarm/security-pack/compare/v0.3.0...v0.3.1) updates kyverno-app to v0.10.1.
- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [v0.0.1](https://github.com/giantswarm/trivy-operator-app/releases/tag/v0.0.1) is the first release for testing `trivy-operator`, which is intended to eventually replace `starboard`. App version 0.0.1 contains upstream version 0.0.5.

```

**Pull request merged**